source/PeekApp.cpp

```diff
@@ -119,9 +120,11 @@ void PeekApp::MessageReceived(BMessage* e) {
 }
 
 void PeekApp::AboutRequested() {
-  BAboutWindow* about = new BAboutWindow(PeekVersion, "application/x-vnd.Peek");
-
-  about->AddText(words->Return(L_BALERT_ABOUT));
+  BAboutWindow* about = new BAboutWindow(PeekVersion, "application/x-vnd.Peek"); 
+  char aboutMsg[255]; 
+  strcpy(aboutMsg,PeekVersion); 
+  about->AddText(words->Return(L_BALERT_ABOUT)); 
+  strcat(aboutMsg, words->Return(L_BALERT_ABOUT));
```

Comment:
Where is aboutMsg used?

Reply:
The version of `PeekApp.cpp` I was using offline to add my changes was a bit outdated, so in my first commit, I accidentally reverted the changes by the commit that updated About Window to use `BAboutWindow`.  I fixed this in my next commit.

Reply:
OK, I see you were using it in BAlert before, but now you copy version to aboutMsg, then add some text to it, and then it gets destroyed when it goes out of scope, without being used at all. It's dead code. Please remove it.

Reply:
I've removed the dead code. I remembered reverting everything to it's original state, however, there may have been problems while pushing.